terraform/certs/docker/client/main.tf

```diff
@@ -6,6 +6,7 @@ variable "private_key" {}
 variable "ca_cert_pem" {}
 variable "validity_period_hours" {}
 variable "early_renewal_hours" {}
+variable "user" {}
```

Comment:
seems like we should provide some defaults for these so we dont have to inject them all the time - 
would provide defaults for ```validity_period_hours``` , ```validity_period_hours``` and ```user```